chain/src/tests/find_fork.rs

```diff
@@ -102,14 +102,14 @@ fn test_find_fork_case2() {
 
     let mut parent = genesis.clone();
     for i in 0..4 {
-        let new_block = gen_block(&parent, i, U256::from(100u64), vec![], vec![]);
+        let new_block = gen_block(&parent, i + 1, U256::from(100u64), vec![], vec![]);
```

Comment:
Why `i` and `i + 1` have changed their positions here and in line 112?

Reply:
SDKs could catch up soon. But I want to know which version would include this change, v0.7.0?

Reply:
Not decided yet, depend on you.

Reply:
I... will rewrite this part.
Should not depend on the hash comparison.
@doitian